recipes/nfir/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain
+from conan.tools.files import copy, get, collect_libs
+from conan.tools.files import apply_conandata_patches
+from os import path
+
+class NFIRConan(ConanFile):
+    name = "nfir"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/usnistgov/NFIR"
+    topics = ("NIST, NFIR")
+    license = "NIST"
+    description = "The NIST Fingerprint Image Resampler NFIR is a library capable of upsampling or downsampling fingerprint images."
+    settings = "os", "compiler", "build_type", "arch"
+    options = { "shared": [True, False] }
+    default_options = { "shared": False }
+    generators = "CMakeDeps"
+    package_type = "library"
+    exports_sources = "CMakeLists.txt", "patches/*"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def requirements(self):
+        self.requires("opencv/4.10.0")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_SHARED"] = self.options.shared
+        tc.variables["USE_NFIMM"] = False
+        tc.variables["_WIN32_64"] = self.settings.os == "Windows"
+        tc.generate()
+
```

Comment:
```suggestion
        deps = PkgConfigDeps(self)
        deps.generate()

```
pkg-config is used to find OpenCV https://github.com/usnistgov/NFIR/blob/b670b673f8c397f37b231d16bc8b41f934d05841/src/lib/CMakeLists.txt#L46

Reply:
just adding this does not work on windows, and it also introduces unnecessary tool requirements